cg/cli/transfer.py

```diff
@@ -36,12 +36,17 @@ def flowcell(context, flowcell_name):
 @click.option(
     "-i", "--include", type=click.Choice(["unset", "not-invoiced", "all"]), default="unset"
 )
+@click.option(
+    "-id", "--sample-id", type=str, default=None, 
```

Comment:
I suggest that you skip the short alternative. It can not use one letter and is easy to mix up with `--include`
Also `type=str` and `default=None` is default so no need to specify them here
```suggestion
    "--sample-id" 
```